artman/conductors/cloudtasks_conductor.py

```diff
@@ -40,21 +46,29 @@ def _pull_and_execute_tasks(task_client, queue_name):
     tasks = pull_task_response.get('tasks', [])
     if not tasks:
         # Sleep for 30 seconds if there is no tasks returned.
-        logger.debug('There is no pending task. Sleep for 30 seconds.')
-        time.sleep(30)
+        logger.debug('There is no pending task. Sleep for 10 seconds.')
+        time.sleep(10)
     for task in tasks:
-        tmp_reporoot, artman_user_config = _prepare_dir()
+        task_id, tmp_root, artman_user_config, log_file_path = _prepare_dir()
+        log_file_handler = None
         try:
+            log_file_handler = _setup_logger(log_file_path)
             logger.info('Starting to execute task %s' % task)
+            if int(task['taskStatus']['attemptDispatchCount']) > MAX_ATTEMPTS:
+                logger.info('Delete task which exceeds max attempts.')
+                _delete_task(task_client, task)
+                continue
             _execute_task(artman_user_config, task)
             _ack_task(task_client, task)
-            logger.info('Task execution finished for %s' % task)
-        except:
-            logger.error('Task execution failed for %s' % task)
+            logger.info('Task execution finished')
+        except Exception as e:
```

Comment:
This will print the result of `str(e)`, which is just the exception message.
That _might_ be fine in this case (perhaps the message is the output of the command that was run?), but I would probably like to be able to have the full traceback also.

See the [traceback](https://docs.python.org/3/library/traceback.html) module for how to do this; most likely you want something like:

```python
logger.error('\n'.join(traceback.format_tb(sys.last_traceback)))
```